docs/promptl/syntax/configuration.mdx

```diff
@@ -17,6 +17,10 @@ top_p: 0.9
 
 PromptL doesn't impose restrictions on what you include in the config section, so you can add any key-value pairs supported by your LLM provider.
 
+<Note>
+  If you're using the Latitude platform, check out the [Latitude Prompt Configuration](/guides/prompt-manager/configuration) guide for a more detailed guide on what you can include in the configuration section.
+</Note>
+
```

Comment:
Since I'm referencing PromptL docs in Latitude's guide, I added this note in PromptL's configuration page because Latitude's guide on the config section is more useful for users that are using our platform.